src/swarm/neo/connection/RequestOnConnBase.d

```diff
@@ -411,9 +411,27 @@ abstract class RequestOnConnBase
             *******************************************************************/
 
             public void add ( T ) ( ref T elem )
+            {
+                this.addPointer(&elem);
+            }
+
+            /*******************************************************************
+
+                Adds a single element to the payload to be sent.
+
+                Note that this takes a pointer to the element, and the element's
+                lifetime needs to match or exceed that of this payload to avoid
+                memory corruption.
+
+                Params:
+                    ptr = pointer to the element to add
```

Comment:
it should be `elem` instead. Otherwise the code needs to be updated

Reply:
Good catch!